playwright/support/fixtures/editor-api.ts

```diff
@@ -45,17 +47,32 @@ export const test = base.extend<EditorApiFixture>({
 					container.style.background = 'white'
 					document.body.appendChild(container)
 
+					const method = type === 'editor' ? 'createEditor' : 'createTable'
 					// @ts-expect-error - OCA.Text is a global
-					await window.OCA.Text.createEditor({
+					await window.OCA.Text[method]({
 						el: container,
 						...(fileId != null ? { fileId } : { content }),
 						readOnly,
 					})
 				},
-				{ containerId, content: options.content, fileId: options.fileId, readOnly: options.readOnly },
+				{ containerId, type: options.type, content: options.content, fileId: options.fileId, readOnly: options.readOnly },
 			)
 		}
 
+		const createEditor = async (
+			options: {
+				type: 'editor' | 'table',
+				content?: string,
+				fileId?: number,
+				readOnly?: boolean,
+			},
+		) => {
+			await createComponent(options)
+		}
+
 		await use(createEditor)
 	},
+	containerId: async (_, use) => {
+		await use('test-editor-api')
+	},
```

Comment:
I think you can just do
```js
containerId: 'test-editor-api',
```

Or if you want to make it customizable with `test.use({ containerId: 'something-else' })`:
```js
  containerId: ['test-editor-api', { option: true }]
```